README.md

```diff
@@ -66,7 +71,8 @@ TIMEZONE=“Etc/GMT+6”
 
 would output:
 
-```Users that contributed between Tue Mar 31 2020 00:00:00 GMT-0600 and Tue Apr 07 2020 23:59:59 GMT-0600 
+```.txt
```

Comment:
It looks like this should display ".txt", but when I view the README it does not. What's going on here?